src/entity/posts/Post.ts

```diff
@@ -149,6 +149,10 @@ export class Post {
   @Column({ default: null })
   visibleAt: Date;
 
+  @Column({ default: null })
+  @Index()
```

Comment:
I think the right index will be a composite index that involves sourceId, pinnedAt, createdAt.
Because this is gonna be they way we will filter on this property. We will check for all posts of a given source id and then sort by pinned and created at

Reply:
Good point. Do we have any pre-existing samples for the composite index? I tried checking the docs I didn't find any relevant steps. Shouldn't having them all with `Index` work, out of the box?

Reply:
Adding `Index` to every column will create separate indexes.
You can add something like this: `@Index('IDX_notification_user_id_created_at', ['userId', 'public', 'createdAt'])`
above the class definition

Reply:
Wow. That's the first time I got to know this. Thanks. I have applied it now.

Do we keep the standalone index of the `pinnedAt` like the others? Or we should remove it?